### DISCUSSION TITLE
How can i make my cat entity sit down?

### DISCUSSION THREAD

**Discussion by Ja90n** [Minestom/Minestom#1801] [Help] [answered]
Hello, I had a question about the minestom API. How can i make my cat entity sit down because Pose,SITTING does not exist in minestom while it does exist in spigot. I there another way to achieve this? Thanks.

**Comment by GreatWyrm** [comment, marked as answer]
Consulting the [entity metatdata](https://wiki.vg/Entity_metadata#Tameable_Animal), you will need to set the cat's metadata to sitting through [TameableAnimalMeta](https://javadoc.minestom.net/net/minestom/server/entity/metadata/animal/tameable/TameableAnimalMeta.html).

**Reply by Ja90n** [reply]
oh did not know i had to do it through entity meta, thanks!
